docs/proposals/20190709-cluster-spec-crds.md

```diff
@@ -0,0 +1,317 @@
+---
+title: Cluster Provider Spec and Status as CRDs 
+authors:
+  - "@pablochacin"
+reviewers:
+  - "@ncdc"
+  - "@vincepri"
```

Comment:
This should match the maintainers list in the OWNER_ALIASES file

Reply:
How exactly should it match? the nicks are identical. By the way, it is copied from the machine bootstrap proposal.

Reply:
The reviewers list should match this list https://github.com/kubernetes-sigs/cluster-api/blob/master/OWNERS_ALIASES#L13-L16

Bootstrap proposal: https://raw.githubusercontent.com/kubernetes-sigs/cluster-api/master/docs/proposals/20190610-machine-states-preboot-bootstrapping.md